pyproject.toml

```
@@ -54,7 +54,11 @@ benchmark = [
 
 
 [project.entry-points.sklearn_engines]
-kmeans = "sklearn_numba_dpex.cluster.kmeans:KMeansEngine"
+kmeans = "sklearn_numba_dpex.kmeans.engine:KMeansSklearnEngine"
```

Let's also remove `Sklearn` from the engine class name.

```suggestion
kmeans = "sklearn_numba_dpex.kmeans.engine:KMeansEngine"
```

is already long enoug.